ansible/configs/sap-hana/pre_software.yml

```diff
@@ -17,7 +17,17 @@
   become: true
   gather_facts: False
   tasks:
-    - command: subscription-manager release --set=8.1
+    - name: Lock RHEL release to SAP supported one
+      redhat_subscription:
+        state: present
+        server_hostname: "{{ set_repositories_satellite_url }}"
+        activationkey: "{{ set_repositories_satellite_activationkey }}"
+        org_id: "{{ set_repositories_satellite_org | default(satellite_org) }}"
+        release: 8.1
```

Comment:
Should this be a variable ? it depend on the image used, right ?
(even if currently this lab only supports 8.1)